bin/drop_filter_results.py

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env python3
+
+import argparse
+import re
+from collections import OrderedDict
+from pathlib import Path
+import csv
+import os
+import pandas as pd
+import pyreadr
+
+
+def FilteringResults(sample, gene_panel, out_drop_ae_rds, out_drop_gene_name, out_drop_as_tsv):
+    # Read gene panel if it has been provided
+    if gene_panel != "None":
+        df_panel = pd.read_csv(gene_panel, sep="\t", skiprows=26)
+
+    if out_drop_ae_rds != "None":
+        rds_AE = pyreadr.read_r(out_drop_ae_rds)
+        df_results_AE = rds_AE[None]
+        # Keep only samples provided to tomte
+        df_results_family_AE = df_results_AE.loc[df_results_AE["sampleID"].isin(sample)]
+        # Count how many events are significant per provided sample
+        df_family_AE_top20 = pd.DataFrame()
```

Comment:
Thinking we should call this something else as the number of events might be above 20 in the case of a large amount of significant results.